boards/arm/nrf52840_pca20035/board.h

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Remove `board.h` (it's deprecated) and include `soc.h` directly, where necessary.